src/polyfills.js

```diff
@@ -0,0 +1 @@
+import 'polyfill-object.fromentries'
```

Comment:
can you make it typescript?

Reply:
Do you mean changing polyfills.js to polyfills.ts?

Reply:
yes 😄

Reply:
ok, It has changed😃